mandatory/3-function-output.js

```diff
@@ -3,15 +3,25 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+//The Math.random() function returns a floating-point, pseudo-random number 
+//that's greater than or equal to 0 and less than 1, with approximately uniform 
+//distribution over that range — which you can then scale to your desired range.
+
```

Comment:
ok, so how do i get a random number between 9 and 20? :)

Reply:
You simply have to multiply function by 20 to get the max variant from 0 to 20,
function getRandomNumber() {
  return Math.random() * 20;
  }
But if you want to make the lowest number 9, you just have to add 9 to the function, but then the multiplier will be 11:
function getRandomNumber() {
 return Math.random() * 11 + 9
 }
 And that's how you do it :)

Reply:
👍